src/frontend/src/pages/WorkPackageForm/EditWorkPackageForm.tsx

```diff
@@ -19,7 +20,7 @@ const EditWorkPackageForm: React.FC<EditWorkPackageFormProps> = ({ wbsNum, setPa
   return (
     <WorkPackageForm
       wbsNum={wbsNum}
-      mutateAsync={mutateAsync}
+      mutateAsync={mutateAsync as MutationFunction}
```

Comment:
instead of casting like this, you should change WorkPackage form to use a correct type (data: PayloadType) => void like we do for the project form

Reply:
Could you explain more about where I make these changes?

Reply:
you'll need to go into the WorkPackageForm component and update its mutateAsync prop to have type `(data:  WorkPackageApiInputs) => void`. That's basically saying it can take in a method with input type   WorkPackageApiInputs (edit work package payload). From there you might need to update workpackageformview and the payload we construct inside of there to get the types to play nice, I'd ask Jack for more specifics about that if you run into problems